src/styles/globals.css

```
@@ -2,151 +2,114 @@
 @tailwind components;
 @tailwind utilities;
 
+@import "./tokens.css";
```

Stylelint (17.12.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)